building/gits/optee_os.rst

```diff
@@ -498,5 +498,158 @@ Some combinations of configuration variables may not be valid. This should be
 dealt with by custom checks in makefiles. ``mk/checkconf.h`` provides functions
 to help detect and deal with such situations.
 
+Import branches
+===============
```

Comment:
This is a separate topic, so probably better to have this as
`*****************`
instead. so it'll be visible to the dropdown in the left menu/tree.

Reply:
Yup, will fix